### PR TITLE
Development into Main

### PR DESCRIPTION
**Pull request by Sisamkele022**
New implementations on my bot

### CODE REVIEW

**Review by Ayanda-Radebe — APPROVED** [review]
good work!

File: .gitignore
Comment (SilindileShabangu) [inline .gitignore:1]: I ADVISE YOU USE THE FIRST (MOVING ) GIF

**Review by SilindileShabangu — APPROVED** [review]

File: .gitignore
Comment (Sisamkele022) [reply .gitignore:1]: I did thank you

**Review by LefaJele-Masemola — APPROVED** [review]
I love the voice automation on the chatbot!

**Review by Lord-Nickelbottoms — APPROVED** [review]
Use an external css file for cleaner code from now on :)

**Comment by Sisamkele022** [comment]
Thank you team for the approvals and suggestions. I appreciate them so much.

**Pull request merged**